### DISCUSSION TITLE
Please add automatic fitting

### DISCUSSION THREAD

**Discussion by lcatalanm** [marp-team/marp#515] [Ideas] [answered]
Sometimes it happens that my text is too long and it oversizes the slide. It would be great to have a way to automatically make it fit.

Please add features like measureText(), fillTextBox() and fitText() from remotion, check this video to understand I mean:
https://www.instagram.com/p/C2PXf73IqF2/

**Comment by yhatt** [comment, marked as answer]
Auto-fitting equivalent to `fitText()` you've mentioned is already available in Marp.
https://github.com/marp-team/marp-core#fitting-header

```markdown
# <!--fit--> Automatic fitting
```
